### DISCUSSION TITLE
「开源之史」系列之九/四：万维网的许可选择——和GPL擦肩而过 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#53] [Announcements]
# 「开源之史」系列之九/四：万维网的许可选择——和GPL擦肩而过 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-06-www-did-not-choose-gpl-and-open-public/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

这其实就是一个壕看不上眼的东西, 被屌丝利用起来变成一门大生意的故事,
CERN 是什么? 搜索一下就知道, 真.壕无双,

现在所谓大规模并行计算之类高端计算机技术在人家那儿是常规操作,
而且只是思考了一下就实现了, 唯一用心的就是连带写了几篇论文而已;

Tim 是真老顽童, 一心只为现实内心想往的世界, 
只是, 因为很早就是终身教授, 在自己的领域中有无数有趣的问题可以研究,
根本没接触过现实抢饭吃的场景,
所以, 对于 gopher/www 都不过是现实目标过程中毫不足道的过程/工具;

但是, 对于资本家, 这可都是钻石矿, 
以往只有拨通电话时, 才能收费, 
现在嘦人活着 , 就有理由收费, 简直不要太幸福...

万维网, 最终只现实了 几维网, 就已经变成万亿级市场, 
其它维可以慢慢商业化, 资本家不傻,

傻的只有 Tim 们, 以为免费分享了, 所有人就可以免费使用,
根本没意识到, 技术免费和最终用户免费中间, 还有无限资本市场隔断着...

这也是为什么一直有断言:

> 商业化, 才是最大的慈善

如果 CERN/Tim 们, 从一开始就垄断了 WWW 市场, 将每年收益投入到研发上,
可能就没硅谷什么事儿了...

**Reply by lijiangsheng1** [reply]
那我们可能过去的30年就完全是另外一个样子了。🤦‍♂️
